pwndbg/disasm/__init__.py

```diff
@@ -119,8 +119,31 @@ def get_disassembler(pc):
                                    pwndbg.arch.endian,
                                    extra)
 
+class SimpleInstruction(object):
```

Comment:
No need to inherit from `object` in Python 3
```suggestion
class SimpleInstruction:
```

Reply:
OK, I'll fix it.